api/src/middleware/auth.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken');
+const config = require('config');
+const secret = config.get('secret');
+
+module.export = (req, res, next) => {
+    const reqHeaders = req.headers('authorisation');
```

Comment:
this middleware will pass through all the requests that don't have JWT, but you should keep in mind the some calls must have `authorisation` header. So make sure to add some kind of validation for the requests that need this header.